packages/expo-file-system-interface/ios/EXFileSystemInterface/EXFilePermissionModuleInterface.h

```diff
@@ -0,0 +1,12 @@
+#ifndef FilePermissionWielderInterface_h
+#define FilePermissionWielderInterface_h
+
+@protocol EXFilePermissionModuleInterface
+
+- (EXFileSystemPermissionFlags)getPathPermissions:(NSString *)path
+                                       scopedDirs:(NSArray<NSString *> *)scopedDirs
+                                  bundleDirectory:(NSString *)bundleDirectory;
```

Comment:
Since an implementation of `EXFilePermissionModuleInterface` will also be an internal module, it will be able to acquire access to an instance of `EXModuleRegistry *` and `EXFileSystemManager`, like `EXFileSystem` does:

https://github.com/expo/expo/blob/068c5923bf412dfb8a2ac0b8f0a38efd8aa0e2b3/packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m#L99

So to sum up, we shouldn't need to pass `bundleDirectory` to this method, it should be inferred by the implementation of `EXFilePermissionModuleInterface`, like here:

https://github.com/expo/expo/blob/068c5923bf412dfb8a2ac0b8f0a38efd8aa0e2b3/packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m#L734